spec/ruby/core/gc/stat_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../spec_helper'
+
+describe "GC.stat" do
+  it "returns zero for unknown key" do
+    keys = [:omganyvalue, :total_allocated_objects]
```

Comment:
In MRI:
```
> GC.stat(:omganyvalue)                                                                                                        
ArgumentError: unknown key: omganyvalue
```

Tests/specs should pass on MRI too.
There are also TruffleRuby-specific specs under `spec/truffle` but I don't think it's needed here.
So I think here we should only test known keys like `:total_allocated_objects`

Reply:
Thanks, fixed.

What's the `jt` command to run the test on MRI to check that it's compatible?

Reply:
it's `jt test path/to/spec.rb -t ruby`, assuming you MRI in your PATH.
We should probably mention that in https://github.com/oracle/truffleruby/blob/master/doc/contributor/workflow.md#testing

Reply:
Thanks!

Updated the test and made sure it's passing on both Truffle and MRI.

Is there anything else you'd like me to address in the PR?